.gitignore

```diff
@@ -65,3 +65,14 @@ test-results/
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+/test-results/
+/playwright-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/playwright/.cache/
```

Comment:
this is repeated 🤔

Reply:
Updated. Could have been lost due to a rebase. Good catch.